common/quotas/ratelimiter.go

```diff
@@ -107,13 +108,13 @@ func (rl *RateLimiter) Allow() bool {
 }
 
 // Limit returns the current rate per second limit for this ratelimiter
-func (rl *RateLimiter) Limit() float64 {
+func (rl *RateLimiter) Limit() rate.Limit {
```

Comment:
Had to change because this is also a `quotas.Limiter`, and the signature is incompatible.